company/team/locations.geojson

```diff
@@ -352,6 +352,19 @@
         ]
       }
     },
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Alex Russell-Saw"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+					51.5085300,
+          -0.1257400
```

Comment:
I think these two need to be swapped, assuming UK. iirc I made the same mistake :sweat_smile: 
```suggestion
			-0.1257400,
			51.5085300
```